modules/UI/feedback/FeedbackWindow.js

```diff
@@ -179,6 +175,9 @@ export default class Dialog {
     }
 
     onHide() {
-        this.onCloseCallback(this.feedbackScore, this.feedbackMessage);
+        this.onCloseCallback({
+            score: this.feedbackScore,
+            message: this.feedbackMessage
+        });
     }
```

Comment:
These parameters do not reach maybeRedirectToWelcomePage as expected by this PR. They are passed to the callback passed to Feedback.openFeedbackWindow, which is only called in UI.requestFeedbackOnHangup.